addons/website_event/static/src/js/website_event.js

```diff
@@ -44,8 +44,8 @@ var EventRegistrationForm = Widget.extend({
         $('#registration_form select').each(function () {
             post[$(this).attr('name')] = $(this).val();
         });
-        var tickets_ordered = Object.values(post).map((value) => parseInt(value))?.length > 0;
-        if (!tickets_ordered) {
+        var no_tickets_ordered = Object.values(post).map((value) => parseInt(value)).every(value => value === 0);
```

Comment:
While we are at it, can we do "const noTicketsOrdered" ?

Better to follow JS guidelines.

Reply:
We could also add a small code comment to make sure no-one breaks this in the future:

`// "post" looks like: { nb_register-9: "2", nb_register-10: "0" }`